ownership-chain/pallets/asset-metadata-extender/src/lib.rs

```diff
@@ -175,6 +175,17 @@ impl<T: Config> AssetMetadataExtender<T> for Pallet<T> {
 		let claimer = Self::claimer_by_index(universal_location.clone(), index)?;
 		TokenUrisByClaimerAndLocation::<T>::get(claimer, universal_location)
 	}
+
+	fn extension_by_location_and_claimer(
+		claimer: AccountIdOf<T>,
```

Comment:
`universal_location` comes first and `claimer` is second param, according to the name of the `function`